packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -33,7 +33,7 @@ export const style = theme => {
     root: {
       position: 'relative',
       width: '100%',
-      margin: '10px 0',
+      margin: '10px',
```

Comment:
Why do we have a margin by default :o.

Reply:
The way the thumb is currently constructed, without it, the active thumb would overlap surrounding components. This was a quick fix for the horizontal overlap.

I guess it's tangentially related to #11889.

Reply:
Should we be using the padding instead? Using the margin is deceptive. One can't override it directly.